src/main/scala/itc/ItcImpl.scala

```diff
@@ -27,7 +28,10 @@ object ItcImpl {
     forUri(Uri.uri("https://gemini-itc.herokuapp.com/json"))
 
   def forUri[F[_]: ConcurrentEffect](uri: Uri): Resource[F, Itc[F]] =
-    AsyncHttpClient.resource[F]().map(forClientAndUri[F](_, uri))
+    AsyncHttpClient.resource[F]()
+      .map(RequestLogger(true, true))
+      .map(ResponseLogger(true, true))
```

Comment:
This is kind of cool. You can add logging via middleware. Right now it logs too much but it's helpful for development. If you want to turn it off this is where it happens.